include/constants.h

```diff
@@ -47,6 +47,7 @@ namespace reinforcement_learning {  namespace name {
       const char *const  DECISION_SEND_QUEUE_MAX_CAPACITY_KB    = "decisions.send.queue.maxcapacity.kb";
       const char *const  DECISION_SEND_BATCH_INTERVAL_MS   = "decisions.send.batchintervalms";
       const char *const  DECISION_SENDER_IMPLEMENTATION    = "decisions.sender.implementation";
+      const char *const  DECISION_RANK_MODE = "decisions.rank.mode";
```

Comment:
Should we fix the naming consistency in a separate PR?
* `DECISION_FOO_BAR = decisions.foo.bar` => `DECISION_FOO_BAR = decision.foo.bar`
* `maxcapacity` => `max.capacity`
* `batchintervalms` => `batch.interval.ms`

Reply:
Also, there is a mismatch in terminology right now.
In client library:
interactions - CB interactions.
decisions - CCB interactions.
Given that your change is only about CB now, renaming "*decision*" to "*interaction*" looks more consistent

Reply:
Renamed them to "Interaction" at this moment.